ui/package.json

```diff
@@ -15,6 +15,8 @@
     "react-router-dom": "^4.2.2"
   },
   "devDependencies": {
+    "ajv": "^6.10.2",
+    "ajv-keywords": "^3.4.1",
```

Comment:
What's requiring the addition of these new dependencies?

Reply:
Good question! Looks like the webpack upgrade is now pulling these two in https://www.npmjs.com/package/webpack?activeTab=dependencies

Reply:
weird that they were added as direct dependencies. I'd think npm would pull them in by nature of webpack having a dependency on them. Are you sure' they're needed here?

Reply:
I think I found out what happened, something else must have a dependency on these because if I get rid of those dependencies, I get this error:

```
./gradlew processResources

> Task :ui:install
npm WARN ajv-errors@1.0.1 requires a peer of ajv@>=5.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN ajv-keywords@3.4.1 requires a peer of ajv@^6.9.1 but none is installed. You must install peer dependencies yourself.
npm WARN ajv-keywords@3.4.1 requires a peer of ajv@^6.9.1 but none is installed. You must install peer dependencies yourself.
```

So it looks like it's more of a version resolution dependency than anything else.

Reply:
@jcohen you OK with this being merged?